Use camlp5 instead of camlp4, which is deprecated and will be removed from Debian. See #933722.
--- a/Makefile
+++ b/Makefile
@@ -61,7 +61,7 @@
 
 CC_CMD = $(COMPILER) $(ALL_CFLAGS) -c $<
 
-CC_PARSER_CMD = $(COMPILER) -pp camlp4o $(ALL_CFLAGS) -c src/syntax/parser.ml
+CC_PARSER_CMD = $(COMPILER) -pp camlp5o $(ALL_CFLAGS) -c src/syntax/parser.ml
 
 RELDIR=../../..
 
